style.css

```diff
@@ -372,71 +362,214 @@ section {
     color: #42526e;
     margin-bottom: 250px;
   }
-}
 
-.contact-us {
-  display: flex;
-  flex-direction: column;
-  align-content: center;
-  justify-content: center;
-  background: url('/images/contact-background-svg.svg');
-  background-size: 5vmax;
-  background-repeat: no-repeat;
-  background-position: center right;
-  background-position-y: 850px;
-}
+  .contact-us {
+    background: url('/images/contact-background-svg.svg');
+    background-repeat: no-repeat;
+    background-position: bottom right;
+  }
 
-p.contact {
-  font-family: 'Inter', sans-serif;
-  font-style: normal;
-  font-weight: 700;
-  font-size: 40px;
-  line-height: 52px;
-  text-align: center;
-  color: #172b4d;
-}
+  .form-body {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    gap: 24px;
+  }
 
-.get-in-touch {
-  display: flex;
-  flex-direction: column;
-  justify-content: center;
-  gap: 24px;
-}
+  .name,
+  .email {
+    box-sizing: border-box;
+    background: #fff;
+    border: 1px solid #d0d9d4;
+    border-radius: 4px;
+    height: 48px;
+    padding-left: 16px;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+    color: #6b778c;
+    width: 350px;
+  }
 
-#name,
-#email {
-  box-sizing: border-box;
-  background: #fff;
-  border: 1px solid #d0d9d4;
-  border-radius: 4px;
-  height: 48px;
-  padding-left: 16px;
-  font-style: normal;
-  font-weight: 400;
-  font-size: 15px;
-  line-height: 24px;
-  color: #6b778c;
-}
+  .message {
+    box-sizing: border-box;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 15px;
+    line-height: 24px;
+    color: #091e42;
+    border: 1px solid #60c095;
+    border-radius: 4px;
+    background: #fff;
+    padding-left: 12px;
+    height: 114px;
+  }
 
-#message {
-  box-sizing: border-box;
-  font-style: normal;
-  font-weight: 400;
-  font-size: 15px;
-  line-height: 24px;
-  color: #091e42;
-  border: 1px solid #60c095;
-  border-radius: 4px;
-  background: #fff;
-  padding-left: 12px;
-  height: 114px;
-}
+  .contact-button {
+    width: 125px;
+    align-self: center;
+    margin-bottom: 55px;
+  }
+
+  footer.social-icons {
+    margin-top: 20px;
+    margin-bottom: 40px;
+  }
 
-.contact-button {
-  width: 125px;
-  align-self: center;
+  /* End of Footer Section */
 }
 
-.contact-us .social {
-  margin-top: 20px;
+/* #### End of Mobile Design #### */
+
+/* #### Start of Laptop Design #### */
+
+@media (min-width: 753px) {
```

Comment:
- [ ] Please note that queries for the Desktop breakpoint in the CSS file (the value for min-width property should be in the range of 768px or wider) otherwise it's gonna be too small to fit the desktop design kindly change `min-width: 753px` to any value between 768 and 1440